client/post-editor/media-modal/detail/_style.scss

```diff
@@ -100,26 +117,43 @@
 	}
 }
 
-.editor-media-modal-detail__edit {
-	font-weight: bold;
+.editor-media-modal-detail__edition-bar {
 	display: none;
+
 	&.is-desktop {
 		@include breakpoint( ">480px" ) {
 			display: block;
 			position: absolute;
-			top: 8px;
-			right: 8px;
+			    top: 8px;
+			    right: 8px;
+
+		    button {
+		    	margin-left: 8px;
+		    }
 		}
 	}
 
 	&.is-mobile {
 		@include breakpoint( "<480px" ) {
 			display: block;
-			width: 100%;
- 			margin-bottom: 16px;
+			button {
```

Comment:
I think we prefer not to use element selectors. Can we use a modifier or a class name?